index.html

```diff
@@ -224,5 +167,40 @@ <h2 class="meetup-date">24 nov</h2>
     </div>
   </footer>
 </body>
+<script>
+fetch('https://api.github.com/repos/devfsa/meetups/issues/3/comments')
+  .then((response)=> {
+      response.json().then((data)=> {
+        data = data.slice(0,4)
+        let html = '';
+        let keynotes = document.getElementById("keynotes");
+        data.forEach(element => {
```

Comment:
I would use a `reduce` and return the html content already built.
```suggestion
        const keynotesHTML = data.reduce((html, keynote) => {
          const { user, url, body, title, reactions } = keynote
          return [
            html,
            '<div class="card keynote-card">',
              '<div class="keynote-image">'
                '<img src="', user.avatar_url, '" class="keynote-photo" />',
              '</div>',
              '<div class="keynote-desc">',
                '<a href=", url, '" class="keynote-title" target="_blank">', title, '</a>',
                '<div class="keynote-info">',
                  '<p class="keynote-author">@', user.login, '</p>',
                  '<a href="', html_url, '" class="keynote-vote voted" target="_blank">',
                    '<i class="far fa-arrow-alt-circle-up"></i>',
                    '<span class="keynote-vote-number">', reactions.length '</span>',
                  '</a>',
                '</div>',
              '</div>',
            '</div>',
          ].join('');
        }, '');
```